### PR TITLE
Sample set 3

### PR DESCRIPTION
**Pull request by anjgola**
I left the robot sample untouched because the setup needs very board specific changes and it is covoluted to update it for up2board. Anyone who wants to change it will anyway have to configure specific pins.

### CODE REVIEW

File: automatic-curtain/cpp/automatic-curtain.cpp [hunk 1]
```diff
@@ -113,6 +88,11 @@ int setup_lux_target(upm::GroveRotary *rotary_sensor,
      * Dividing Rotary_sensor->abs_deg() by 6 interval [0-60] is
      * obtained.
      */
+#ifdef SIMULATE_DEVICES
```
Comment (shahabl) [inline automatic-curtain/cpp/automatic-curtain.cpp]: This should be #ifndef

File: automatic-curtain/cpp/automatic-curtain.cpp [hunk 2]
```diff
@@ -268,8 +299,13 @@ int main() {
   // Loop forever
   for (;;) {
 
+#ifndef SIMULATE_DEVICES
```
Comment (shahabl) [inline automatic-curtain/cpp/automatic-curtain.cpp]: This should be #ifdef

File: aws-pub-sub/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# AWS® IOT Pub-Sub Sample
```
Comment (shahabl) [inline aws-pub-sub/README.md]: There's already a README for this sample in cpp folder

**Comment by anjgola** [comment]
Updated as per review comments

**Pull request merged**